### PR TITLE
Remove ripple effect when tapping in the middle

### PR DESCRIPTION
**Pull request by ricardodalarme**
## Description

- Remove the ripple effect that occurred when tapping in the middle of the screen to match the same behavior when tapping to the side.
- Also include changes to the SeekHandler:
  - Do not pass the exoPlayer instance since it is unstable
  - Remove global jobs holder
  - Enable instant forward when rewinding and vice versa

### Pull request checklist

- [ ] **Tests**: This PR includes tests for covering the features or bug fixes (if applicable).
- [ ] **Docs**: This PR updates/creates the necessary documentation.
- [x] **CI**: Make sure your Pull Request passes all CI checks. If not, clarify the motif behind that and the action plan to solve it (may reference a ticket)

## How to test it

- Tap in the middle of the screen

#### Before

https://github.com/profusion/android-enhanced-video-player/assets/35314270/d691509c-f2de-4acd-8b9c-0fde46b939dc

#### After

https://github.com/profusion/android-enhanced-video-player/assets/35314270/fb6343eb-2997-4ac1-9b8d-e8a7187cbbef

### CODE REVIEW

**Review by loadez — APPROVED** [review]
LGTM. Perhaps the improvement of the seeking function should me mentioned on this PR

**Comment by ricardodalarme** [comment]
Thanks, @loadez. I forgot to describe these changes because they were introduced after I opened the PR. Nice catch!

File: androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/utils/ExoPlayerUtils.kt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.profusion.androidenhancedvideoplayer.utils
+
+import androidx.media3.exoplayer.ExoPlayer
+
+fun ExoPlayer.seekIncrement(incrementMs: Long) {
```
Comment (thalysmcarrara) [inline androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/utils/ExoPlayerUtils.kt:5]: nice!

File: androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/utils/CustomModifiers.kt [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+import androidx.compose.ui.composed
 
 fun Modifier.fillMaxSizeOnLandscape(orientation: Int) =
     when (orientation) {
         Configuration.ORIENTATION_LANDSCAPE -> this.then(Modifier.fillMaxSize())
         else -> this
     }
+
+fun Modifier.noRippleClickable(enabled: Boolean = true, onClick: () -> Unit): Modifier = composed {
+    clickable(
```
Comment (thalysmcarrara) [inline androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/utils/CustomModifiers.kt:18]: I think this component is not necessary as it is only being used in one place. do u have another usage in mind?

**Review by thalysmcarrara — APPROVED** [review]
pre approving

File: androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/utils/CustomModifiers.kt (hunk 2 above)
Comment (ricardodalarme) [reply androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/utils/CustomModifiers.kt:18]: Nope, I don't, but I keep it to make it simpler to create a clickable without the ripple effect

**Pull request merged**